gatsby-config.js

```diff
@@ -357,6 +391,40 @@ module.exports = {
                           },
                         ],
                       },
+                      {
+                        label: "Equity Embed",
+                        name: "equityEmbed",
+                        widget: "object",
+                        fields: [
+                          {
+                            label: "Attribution",
+                            name: "attribution",
+                            widget: "string",
+                          },
+                          {
+                            label: "Image",
+                            name: "image",
+                            allow_multiple: false,
+                            media_folder: "../../images/quotes",
```

Comment:
We should be pointing to the same folder as in the Main Page (just `src/images/quotes/`

Reply:
I tried that, but it seemed to cause a bug with the hook that finds the images.

Reply:
Ah, hm. Probably because we prefix the image path in that hook, and the State Page's content page is nested?

Reply:
I think that was the issue. when i updated all the media folders to "src/images/quote" in the gatsby-config.js file the array had it as "../../etc" while the image was "src/images/quote/etc." So I changed it back. Let me know if you want me to spend more time on this.

Reply:
This is fine. I am just wondering about the extra `src/pages/state/src/images/quotes/equity-state.jpg` file I see, and want to make sure it's not related?